eva_submission/eload_submission.py

```diff
@@ -57,6 +54,72 @@ def _get_dir(self, key):
     def now(self):
         return datetime.now()
 
+    def upgrade_config_if_needed(self, analysis_alias=None):
```

Comment:
This is a good function to have but I think the core of the upgrade should be a in a separate file similar to how database migration are done i.e. `upgrade_version_0_1.py`
Then this function will only select which upgrade needs to be applied.